### ISSUE TITLE
Authentication

### ISSUE THREAD

**Issue by nrg500** [wmeints/rest-services#3]
The guideline leaves out authentication entirely at the moment, I would argue it is so common that we should include it.
I would add:
401 Not Authorized: User is not authorized properly
403 Forbidden: User does not have the proper priviliges
To all the requests.
Perhaps a small general part that applies to all requests should be added.

**Comment by wmeints** [comment]
I agree. I haven't gotten around to writing this down. I'm thinking of a separate section that describes API security as well. We have some parts readily available at my current project so we can include that.
